controllers/result.go

```diff
@@ -129,6 +133,20 @@ func (rc *ResultController) getResultID() (int64, error) {
 	return resultID, nil
 }
 
+func (rc *ResultController) queryFromParams() map[string]interface{} {
+	searcheKeyword := rc.GetString("keyword")
```

Comment:
```suggestion
	searchKeyword := rc.GetString("keyword")
```

Reply:
@gutakk Fixed in [60c8868](https://github.com/carryall/go-google-scraper-challenge/pull/74/commits/60c88680ce0842bab23fee76366f3d462e18e19c)